test/conformance/tests/test_sriov_operator.go

```diff
@@ -1188,7 +1188,7 @@ var _ = Describe("[sriov] operator", func() {
 						capacity, _ = resNum.AsInt64()
 						res["openshift.io/testresource1"] = capacity
 						return res
-					}, 2*time.Minute, time.Second).Should(Equal(map[string]int64{
+					}, 15*time.Minute, time.Second).Should(Equal(map[string]int64{
```

Comment:
why this change needed ? arent we just moving things around and defining interfaces?

Reply:
yep that was just because there was some flakes in the virtual CI not related to this PR